app/src/main/java/com/jerboa/feat/BlurNsfwTypes.kt

```diff
@@ -0,0 +1,27 @@
+package com.jerboa.feat
+
+import android.os.Parcelable
+import androidx.annotation.StringRes
+import com.jerboa.R
+import com.jerboa.datatypes.types.PostView
+import kotlinx.parcelize.Parcelize
+
+@Parcelize
+enum class BlurNsfwTypes(@StringRes val resId: Int) : Parcelable {
+    Nothing(R.string.app_settings_nothing),
+    All(R.string.account_settings_all),
+    OnlyFromNsfwCommunity(R.string.app_settings_blur_only_from_nsfw_community),
+    OnlyFromSfwCommunity(R.string.app_settings_blur_only_from_sfw_community),
+}
+
+fun BlurNsfwTypes.needNsfwBlur(postView: PostView) =
```

Comment:
These two functions you can probably rename to just `blur` or `isBlurred` . See comment below for the options